http/grpc/src/test/java/io/quarkus/ts/http/grpc/GrpcMtlsEncryptedPemTlsRegistryIT.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.ts.http.grpc;
+
+import static io.quarkus.test.services.Certificate.Format.ENCRYPTED_PEM;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.GrpcService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Certificate;
+import io.quarkus.test.services.Certificate.ClientCertificate;
+import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.ts.grpc.GreeterGrpc;
+import io.quarkus.ts.grpc.HelloReply;
+import io.quarkus.ts.grpc.HelloRequest;
+
+@QuarkusScenario
+public class GrpcMtlsEncryptedPemTlsRegistryIT {
+
+    private static final String CERT_PREFIX = "grpc-mtls-separate-server";
+    private static final String CLIENT_CN_NAME = "mtls-client-name";
+    @QuarkusApplication(grpc = true, ssl = true, certificates = @Certificate(prefix = CERT_PREFIX, clientCertificates = {
+            @ClientCertificate(cnAttribute = CLIENT_CN_NAME) }, format = ENCRYPTED_PEM, configureKeystore = true, configureTruststore = true, tlsConfigName = "mtls-server", configureHttpServer = true)
+
+    )
```

Comment:
Any reason for these two lines? Why not move the closing bracket to line 23?

Reply:
nope, I've just tweaked it, thanks.